ui/components/DataTable.tsx

```diff
@@ -208,11 +208,15 @@ function UnstyledDataTable({
 
 export const DataTable = styled(UnstyledDataTable)`
   width: 100%;
+  padding-right: ${(props) => props.theme.spacing.medium};
   h2 {
+    padding: 0;
     font-size: 18px;
     font-weight: 600;
     color: ${(props) => props.theme.colors.neutral30};
     margin: 0px;
+    text-overflow: ellipsis;
```

Comment:
Which columns are overflowing? There may be a bigger problem happening if we have overflowing column headings.

Reply:
I added `nowrap` for the header row so that Last Updated wouldn't wrap - It wasn't ever going to the ellipsis for me but I put it there just in case.